website/index.html

```diff
@@ -103,12 +103,12 @@ <h2 id="example">Example <a href="#example">#</a></h2>
 
       <p>Or a more complex one:</p>
 
-      <pre><code class="typescript language-typescript">import { serve } from "https://deno.land/x/http/server.ts";
+      <pre><code class="typescript language-typescript">import { serve } from "https://deno.land/x/std/http/server.ts";
 const s = serve("0.0.0.0:8000");
 
 async function main() {
-  for await (const req of s) {
-    req.respond({ body: new TextEncoder().encode("Hello World\n") });
+  for await (const { req, res } of s) {
```

Comment:
No need to destructure `req` as it is not used.

```suggestion
  for await (const { res } of s) {
```

Reply:
Yes, you're right though I thought it might be a good idea to show the `req` there too so that it would look more familiar to node people so they can see how both `req` and `res` can be used. But I can update it to only destructure `res` if that's the final decision?

Just going to wait for confirmation since it was mentioned that there shouldn't be much iteration for trivial doc/css stuff.

Reply:
If you aren't using it, you aren't showing anything, except that you can create extra variables that aren't being used.

Reply:
Sorry, to clarify – I meant that people who are coming from stuff like node who are used to `(req, res) => { ... }` might wonder how they can access the `req` object in deno when they look at the http server example if it just had `res`, so I thought it would be a good idea to show the variable there, but I suppose it might be obvious that it could be destructured as well. Removing it as requested.